controllers/nstemplateset/status_test.go

```diff
@@ -51,6 +50,58 @@ func TestUpdateStatus(t *testing.T) {
 			HasConditions(condition)
 	})
 
+	t.Run("update provisioned namespaces", func(t *testing.T) {
+		// given
+		conditions := []toolchainv1alpha1.Condition{{
+			Type:   toolchainv1alpha1.ConditionReady,
+			Status: corev1.ConditionTrue,
+		}}
+		nsTmplSet := newNSTmplSet(namespaceName, username, "basic", withNamespaces("abcde11", "dev", "stage"), withConditions(conditions...))
```

Comment:
Maybe
```suggestion
		nsTmplSet := newNSTmplSet(namespaceName, username, "basic", withNamespaces("abcde11", "stage", "dev"), withConditions(conditions...))
```
to make sure the sorting kicks in?

Reply:
Good point! But I did it a bit different here, since those variables were not actually used. I've inverted the order also in `nstemplateset_controller_test.go`.

See [b29f373](https://github.com/codeready-toolchain/member-operator/pull/407/commits/b29f373ef5e18d09e056dab4d74ed7dc7236ae95)